cmd/kubeadm/app/phases/certs/doc.go

```diff
@@ -36,6 +36,12 @@ package certs
 		 - apiserver.key
 		 - apiserver-kubelet-client.crt
 		 - apiserver-kubelet-client.key
+		 - etcd.crt
```

Comment:
You also need to update `INPUTS` to include EtcdCertSANs.

Reply:
Thanks. I've updated the doc-strings for both the SAN components.